eng/pipelines/coreclr/templates/build-corelib-job.yml

```diff
@@ -82,7 +82,7 @@ jobs:
       - template: /eng/pipelines/common/restore-internal-tools.yml
 
     # Build Private CoreLib
-    - script: $(Build.SourcesDirectory)$(dir)build$(scriptExt) -subset clr.corelib $(crossArg) -arch $(archType) -c $(buildConfig) $(officialBuildIdArg) -ci
+    - script: $(Build.SourcesDirectory)$(dir)build$(scriptExt) -subset clr.corelib $(crossArg) -arch $(archType) $(osArg) -c $(buildConfig) $(officialBuildIdArg) -ci
```

Comment:
PR: https://github.com/dotnet/runtime/pull/34664 is removing this template as we will just build corelib as part of the libraries job, so I suggest rebasing this PR once that is merged which is pretty much ready to go. cc: @mangod9

Reply:
I have just merged, so you can pull those changes which should remove this template. Thx.

Reply:
updated & testing.